### ISSUE TITLE
What is AbioticAlice?

### ISSUE THREAD

**Issue by jMyles** [nucypher/nucypher#2754]
`Alice`, we understand increasingly, is the singular and final authority over all `Policies` which are governed by the `DelegatingPower` whose secret she keeps in the course of her keystore diligence.  We have occasionally regarded `Alice` as a "data owner", but I have often pushed back on this as too narrow to be useful in some situations.  In particular, regarding Alice as the data owner seemingly means, at a minimum, that she has read access to the data at some moment in time.  And while this is true for all of the Alices we've made so far, I think it is actually a coincidence of the type of keystore diligence we have understood as coupled to `DelegatingPower`.

Specifically, we have presumed that the diligence is exercised by one node (belonging to the user who has initiated the `Alice`, and out of whose wallet the `Policy` is funded), where the single, centralized delegating secret is custodied.  I move that, henceforth, we call this type of keystore diligence **centralized keystore diligence**, for it is *centered* around the one keystore where the delegating secret resides.  All of the `Policies`, `Bobs`, `TreasureMaps`, `KFrags`, and other cryptographic consequences of `grant` and `revoke` orbit around this centralized keystore.

`AbioticAlice` is a set of nodes with `DelegatingPower` which exercise **decentralized keystore diligence**, in contrast to the centralized diligence of an `AnimateAlice` (a turn of phrase).

This means that, instead of a single Character mapping to a single node, a cohort of nodes cultivate together the keystore diligence required to exercise the `DelegatingPower` which governs `AbioticPolicies`, the granting and revocation of which can be triggered conditionally as a contractual matter (#2758) or pursuant to qualified network activity (#2757).

`AbioticAlice` is a highly ritualistic `Character`.  She appears not by the execution of a single command in a single CLI, but instead as the result of the **Ritual to Summon AbioticAlice** (#2755), which binds a `Clique` of nodes together to act as `Colleagues` of this **decentralized keystore diligence**.  Her activities, such as `grant` and `revoke`, are the subjects of subsequent `Rituals`, performed by the same `Clique`.  In the lawful case, there is probably no activity that `AbioticAlice` can perform by command, only by `Ritual`.  An `AbioticAmonia` may demonstrate the folly of attempting to exercise the same decentralized `DelegatingPower` by command alone.

**Closing condition**: The creation of an `AbioticAlice` class, with logic for **decentralized keystore diligence**.

**Comment by cygnusv** [comment]
I'm surprised you didn't follow the `BorgAlice` direction.

**Comment by jMyles** [comment]
Had it not been for the emergence of the Borg Queen, which I thought really kinda ruined their stoic nature, I might have.  ;-)

**Comment by cygnusv** [comment]
This is a very good issue, and it will be a huge milestone to close it. I added some comments in the other issues you opened. My only comment here is regarding the name. I wouldn't mind keeping `Azoic`, but I want to propose `Abiotic`:
```
abiotic | ˌeɪbʌɪˈɒtɪk |
adjective
physical rather than biological; not derived from living organisms: abiotic chemical reactions.
• devoid of life; sterile: soils are seldom completely abiotic.
```

* It starts with an A, which is what I assume you were going for with `Azoic`
* It also carries the meaning of "_no life_"
* Fairly natural etymology, derived from classic Greek (although, to be fair, `Azoic` has the same origin, but for some reason this wasn't my first intuition)

**Comment by jMyles** [comment]
Love it.

**Comment by KPrasch** [comment]
Closed by #3105
